services/sidecar/src/simcore_service_sidecar/celery.py

```diff
@@ -28,7 +44,7 @@ def worker_shutting_down_handler(
 
 @worker_ready.connect
 def worker_ready_handler(*args, **kwargs):  # pylint: disable=unused-argument
-    log.info("!!!!!!!!!!!!!! Worker is READY now !!!!!!!!!!!!!!!!!!")
+    print(WELCOME_MSG)
```

Comment:
MINOR: add ``python(WELCOME_MSG, flush=True)`` to ensure it marks the moment the server has started

Reply:
done